docker/docker.go

```diff
@@ -25,6 +26,11 @@ const (
 )
 
 func main() {
+	// Set terminal emulation based on platform as required.
+	stdout, stderr, stdin := term.StdStreams()
+
+	initLogging(stderr)
```

Comment:
Just a question, but should be keep `reexec.Init()` the first thing? Some of the use of `reexec` being security related, I'm wondering if it wouldn't be safer to call the less possible code between `main()` invocation and actual `reexec`.

Any thoughts @tiborvass?